packages/playground/_stories/main/Table/Table.stories.ts

```diff
@@ -24,1133 +29,979 @@ export default {
-				<span>Supplier</span>
-			</ui5-table-column>
-			<ui5-table-column slot="columns" min-width="800" popin-text="Dimensions" demand-popin="" class="table-header-text-alignment">
-				<span>Dimensions</span>
-			</ui5-table-column>
-			<ui5-table-column slot="columns" min-width="800" popin-text="Weight" demand-popin="" class="table-header-text-alignment">
-				<span>Weight</span>
-			</ui5-table-column>
-			<ui5-table-column slot="columns" class="table-header-text-alignment">
-				<span>Price</span>
-			</ui5-table-column>
-		</ui5-table>
-	</div>
-`;
+};
+TableNoData.storyName = "Table with No Data";
 
+export const GrowingTableMoreButton= Template.bind({});
+GrowingTableMoreButton.args = {
+	growing: TableGrowingMode.Button,
+	growingButtonSubtext: "[4 / 11]",
+	columns: `
+<ui5-table-column slot="columns">
+	<span style="line-height: 1.4rem">Product</span>
+</ui5-table-column>
+<ui5-table-column slot="columns" min-width="800">
+	<span style="line-height: 1.4rem">Supplier</span>
+</ui5-table-column>
+<ui5-table-column slot="columns" min-width="600" popin-text="Dimensions" demand-popin="" class="table-header-text-alignment">
+	<span style="line-height: 1.4rem">Dimensions</span>
+</ui5-table-column>
+<ui5-table-column slot="columns" min-width="600" popin-text="Weight" demand-popin="" class="table-header-text-alignment">
+	<span style="line-height: 1.4rem">Weight</span>
+</ui5-table-column>
+<ui5-table-column slot="columns" class="table-header-text-alignment">
+	<span style="line-height: 1.4rem">Price</span>
+</ui5-table-column>`
 
-export const Template3: StoryFn = () => html`
-<h3>Table Display Inline</h3>
-	<div class="snippet flex-column">
-		<!-- Header -->
-		<div class="header">
-			<span>Products table - resize your browser to make the columns display inline</span>
-			<button id="toggleSticky" style="height: 32px">Toggle Sticky Column Header</button>
-		</div>
-		<ui5-table class="demo-table" id="tblDisplayInline">
-			<!-- Columns -->
-			<ui5-table-column slot="columns" popin-display="Inline">
-				<span>Product</span>
-			</ui5-table-column>
-			<ui5-table-column slot="columns" min-width="600" popin-text="Supplier" demand-popin="" popin-display="Inline">
-				<span>Supplier</span>
-			</ui5-table-column>
-			<ui5-table-column slot="columns" min-width="800" popin-text="Dimensions" demand-popin="" class="table-header-text-alignment" popin-display="Inline">
-				<span>Dimensions</span>
-			</ui5-table-column>
-			<ui5-table-column slot="columns" min-width="800" popin-text="Weight" demand-popin="" class="table-header-text-alignment" popin-display="Inline">
-				<span>Weight</span>
-			</ui5-table-column>
-			<ui5-table-column slot="columns" class="table-header-text-alignment" popin-display="Inline">
-				<span>Price</span>
-			</ui5-table-column>
-		</ui5-table>
-	</div>
-`;
+};
+GrowingTableMoreButton.decorators = [
+	(story) => {
+		return html`
+${story()}
+<script>
+	// Growing Table
+	var products = {
```

Comment:
const instead of vars, in the past when we used to support ie11, we had to use vars, now const is fine

Reply:
Done